### ISSUE TITLE
Error code 65 returned when attempting to run command: /usr/bin/env xcrun xcodebuild -target Runner -configuration Release -sdk iphoneos -arch arm64

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#1949]
```
ianh-macbookpro:~ ianh$ mkdir dev
ianh-macbookpro:~ ianh$ cd dev
ianh-macbookpro:dev ianh$ git clone https://github.com/flutter/flutter.git -b master
Cloning into 'flutter'...
remote: Counting objects: 35943, done.
remote: Compressing objects: 100% (1232/1232), done.
remote: Total 35943 (delta 802), reused 0 (delta 0), pack-reused 34699
Receiving objects: 100% (35943/35943), 12.76 MiB | 2.25 MiB/s, done.
Resolving deltas: 100% (23736/23736), done.
Checking connectivity... done.
ianh-macbookpro:dev ianh$ flutter create -o story
Downloading Dart SDK 1.14.1...
######################################################################## 100.0%
Building flutter tool...
Creating story...
  story/.analysis_options
  story/.gitignore
  story/README.md
  story/android/AndroidManifest.xml
  story/android/res/.empty
  story/flutter.yaml
  story/ios/Assets.xcassets/AppIcon.appiconset/Contents.json
  story/ios/Info.plist
  story/ios/LaunchScreen.storyboard
  story/lib/main.dart
  story/pubspec.yaml

Running 'pub get' in 'story'...

All done! To run your application:

  $ cd story
  $ flutter run

ianh-macbookpro:dev ianh$ cd story
ianh-macbookpro:story ianh$ flutter --verbose run
[  44 ms] - Unable to locate an Android SDK.
[  57 ms] - Downloading toolchain.
[ 111 ms] - Unable to locate an Android SDK.
[ 116 ms] - /usr/local/bin/adb devices -l
[ 128 ms] - List of devices attached
[ 130 ms] - which idevice_id
[ 134 ms] - /usr/local/bin/idevice_id
[ 134 ms] - /usr/local/bin/idevice_id -l
[ 141 ms] - c4ca6f7a53027d1b7e4972e28478e7a28e2faee2
[ 141 ms] - which ideviceinfo
[ 145 ms] - /usr/local/bin/ideviceinfo
[ 146 ms] - /usr/local/bin/ideviceinfo -k DeviceName -u c4ca6f7a53027d1b7e4972e28478e7a28e2faee2
[ 215 ms] - iPod touch
[ 215 ms] - which ideviceinstaller
[ 219 ms] - /usr/local/bin/ideviceinstaller
[ 219 ms] - which idevicedebug
[ 223 ms] - /usr/local/bin/idevicedebug
[ 223 ms] - which idevicesyslog
[ 227 ms] - /usr/local/bin/idevicesyslog
[ 227 ms] - which ios-deploy
[ 230 ms] - /usr/local/bin/ios-deploy
[ 232 ms] - /usr/bin/xcrun simctl list --json devices
[ 305 ms] Downloading darwin-x64 artifacts from the cloud, one moment please...
[ 309 ms] Downloading https://storage.googleapis.com/mojo_infra/flutter/9e1e1fba5474b458458b65f1aa1d0d730af2705b/darwin-x64/artifacts.zip.
[ 599 ms] - Received response statusCode=200
[2180 ms] - Running build command.
[2181 ms] - /usr/local/bin/idevice_id -l
[2189 ms] - c4ca6f7a53027d1b7e4972e28478e7a28e2faee2
[2189 ms] - Running stop command.
[2190 ms] - Running install command.
[2190 ms] - /usr/local/bin/idevice_id -l
[2198 ms] - c4ca6f7a53027d1b7e4972e28478e7a28e2faee2
[2198 ms] - Running build command for IOSDevice c4ca6f7a53027d1b7e4972e28478e7a28e2faee2.
[2199 ms] Starting lib/main.dart on iPod touch...
[2200 ms] - /usr/local/bin/idevice_id -l
[2207 ms] - c4ca6f7a53027d1b7e4972e28478e7a28e2faee2
[2208 ms] - /usr/local/bin/idevice_id -l
[2217 ms] - c4ca6f7a53027d1b7e4972e28478e7a28e2faee2
[2218 ms] - /usr/local/bin/ideviceinstaller --list-apps
[3789 ms] - Total: 0 apps
[3791 ms] - /usr/local/bin/ideviceinstaller -i ios/.generated
[5365 ms] - Building .generated for c4ca6f7a53027d1b7e4972e28478e7a28e2faee2
[5368 ms] - A revision stamp does not exist. The Xcode project has never been initialized.
[5368 ms] - Initializing the Xcode project.
[5370 ms] Fetching the Xcode project archive from the cloud...
[5370 ms] Downloading https://storage.googleapis.com/flutter_infra/flutter/9e1e1fba5474b458458b65f1aa1d0d730af2705b/ios/FlutterXcode.zip...
[14097 ms] Unzipping Xcode project to local directory...
[14223 ms] - /bin/rm -rf /Users/ianh/dev/story/ios/.generated
[14228 ms] - /bin/mkdir -p /Users/ianh/dev/story/ios/.generated
[14232 ms] - /usr/bin/unzip /var/folders/00/03kph000h01000cxqpysvccm000f_t/T/FlutterXcode.zip -d /Users/ianh/dev/story/ios/.generated
[15271 ms] - Archive:  /var/folders/00/03kph000h01000cxqpysvccm000f_t/T/FlutterXcode.zip
   creating: /Users/ianh/dev/story/ios/.generated/Flutter/
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Flutter.xcconfig  
   creating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication/
 extracting: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication/.gitignore  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication/FlutterApplication.c  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication/FlutterApplication.h  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication/Info.plist  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication/InstructionsSnapshotSource.S  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication/kDartIsolateSnapshotBufferSource.c  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication/kDartVmIsolateSnapshotBufferSource.c  
   creating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication.xcodeproj/
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication.xcodeproj/project.pbxproj  
   creating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication.xcodeproj/project.xcworkspace/
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication.xcodeproj/project.xcworkspace/contents.xcworkspacedata  
   creating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication.xcodeproj/xcshareddata/
   creating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication.xcodeproj/xcshareddata/xcschemes/
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication.xcodeproj/xcshareddata/xcschemes/Application.xcscheme  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication.xcodeproj/xcshareddata/xcschemes/Framework.xcscheme  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Local.xcconfig  
   creating: /Users/ianh/dev/story/ios/.generated/Flutter/Runner/
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Runner/main.m  
   creating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/
   creating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/common/
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/common/EmbedderEntryPoints  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/common/icudtl.dat  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/common/PackagerInvoke  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/common/ScriptSnapshotter  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/common/SnapshotterInvoke  
   creating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/iphoneos/
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/iphoneos/FlutterRunner  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/iphoneos/Snapshotter  
   creating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/iphonesimulator/
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/iphonesimulator/FlutterRunner  
  inflating: /Users/ianh/dev/story/ios/.generated/Flutter/Tools/iphonesimulator/Snapshotter
[15271 ms] - /bin/rm -rf /var/folders/00/03kph000h01000cxqpysvccm000f_t/T
[15279 ms] - /bin/mv /Users/ianh/dev/story/ios/.generated/Flutter/Flutter.xcconfig /Users/ianh/dev/story/ios/.generated
[15284 ms] - /bin/mv /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication /Users/ianh/dev/story/ios/.generated
[15287 ms] - /bin/mv /Users/ianh/dev/story/ios/.generated/Flutter/FlutterApplication.xcodeproj /Users/ianh/dev/story/ios/.generated
[15291 ms] - /bin/mv /Users/ianh/dev/story/ios/.generated/Flutter/Local.xcconfig /Users/ianh/dev/story/ios/.generated
[15295 ms] - /bin/mv /Users/ianh/dev/story/ios/.generated/Flutter/Runner /Users/ianh/dev/story/ios/.generated
[15299 ms] - /bin/mv /Users/ianh/dev/story/ios/.generated/Flutter/Tools /Users/ianh/dev/story/ios/.generated
[15303 ms] - /bin/rm -rf /Users/ianh/dev/story/ios/.generated/Flutter
[15312 ms] Xcode project created at /Users/ianh/dev/story/ios/.generated/.
[15312 ms] User editable settings are in /Users/ianh/dev/story/ios/.
[15313 ms] - sky_engine revision: 9e1e1fba5474b458458b65f1aa1d0d730af2705b, iOS engine revision: 9e1e1fba5474b458458b65f1aa1d0d730af2705b
[15314 ms] - xcodebuild -version
[15422 ms] - Xcode 7.2.1
Build version 7C1002
[15424 ms] - Trying to resolve native pub services.
[15425 ms] - Looking for YAML at 'flutter.yaml'
[15460 ms] - No services specified in the manifest
[15460 ms] - Found 0 service definition(s).
[15461 ms] - Copying service frameworks to '/Users/ianh/dev/story/ios/.generated/Frameworks'.
[15462 ms] - Creating service definitions manifest at 'ios/.generated/ServiceDefinitions.json'
[15465 ms] - /usr/bin/env xcrun xcodebuild -target Runner -configuration Release -sdk iphoneos -arch arm64
[16368 ms] - Error code 65 returned when attempting to run command: /usr/bin/env xcrun xcodebuild -target Runner -configuration Release -sdk iphoneos -arch arm64
[16370 ms] ** BUILD FAILED **


The following build commands failed:
    Check dependencies
(1 failure)
[16371 ms] Could not build the precompiled application for the device.
[16371 ms] Error starting application on iPod touch.
[16371 ms] - /usr/local/bin/idevice_id -l
[16379 ms] - c4ca6f7a53027d1b7e4972e28478e7a28e2faee2
[16379 ms] Unable to run application.
[16380 ms] - Finished start command.
ianh-macbookpro:story ianh$ 
```

**Comment by abarth** [comment]
@chinmaygarde

**Comment by Hixie** [comment]
This was with an iPod Touch plugged in. I have never payed Apple for a certificate, so maybe that's the root cause?

**Comment by cbracken** [comment]
This should be fixed by #7879.
